src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -980,6 +980,30 @@ void Convolution::createDescriptor(const std::vector<MemoryDescPtr>& inputDesc,
         memory::data_type bdt = outDnnlDesc.get_data_type();
 #else
         memory::data_type bdt = memory::data_type::f32;
+        /* brdgmm_dw_conv has more perf gain on bf16/fp16 inference.
```

Comment:
@liubo-intel Could you please create ticket on oneDNN with corresponding feature request and put the number here?
There is still some minor accuracy impact and would be great to extend support matrix on the backend side.

Reply:
yeah, if oneDNN can extend support this kind of kernel, I think we can both keep the performance gain and without accuracy impact. have created the request ticket.